test/Index/multifile.swift

```diff
@@ -0,0 +1,13 @@
+// RUN: %empty-directory(%t)
+// RUN: split-file %s %t
+
```

Comment:
Could you add a comment for what this test is checking? And what do we actually end up outputting for the `Bar` references below?

Reply:
done